SPECS/kata-containers/runtime-reduce-uvm-high-mem-footprint.patch

```diff
@@ -17,22 +17,26 @@ current mechanism of requesting resources through pod Limit annotations:
   ContainerCreating state
 
 Testing: Manual testing, creating pods with Limits and without limits, and with two containers where each container has a limit, tested with integration in a SPEC file where the config variables were set via environment variables via the make command
+
```

Comment:
I have manually compared the current (CBL-Mariner/main) and new patch file (this branch/PR) against 3.1 and against 3.2. The changes look good. This patch has been properly ported.